Llaitun.py

```diff
@@ -110,6 +138,20 @@ def live():
         print(ex)
         print(colored("run as super user.", "red"))
 
+def activeHosts(network):
+    check_root()
+    check_net = False
+    while check_net == False:
+        try:
+            ipaddress.ip_network(network, False)
+            check_net = True
+        except ValueError:
+            print(colored("Enter a valid network", "red"))
+            network = input(colored("Network: ", "green"))
```

Comment:
Este script no me queda tan claro, 
por ejemplo no cacho que pasa con ese while que está ahí. 
No le veo mucha funcionalidad, dado que en el try y en el except verificaras la variable "network"
yo sacaría el while, dejaría solamente el try y el except. En el except haría un "exit(1)" luego del mensaje de error.

Reply:
def activeHosts(network):
    check_root()
    try:
        ipaddress.ip_network(network, False)
    except ValueError:
        print(colored("Invalid network", "red"))
        exit(1)
    print(colored("\n[*] Active hosts:", "red"))
    hosts = scapy.all.arping(network,verbose=0)
    for i in hosts[0]:
        print(colored("[+] MAC: ", "green") + i[1].src + colored(" IP: ", "green") + i[1].psrc)